README.md

```diff
@@ -133,6 +137,75 @@ Assets and events will ignore existing ids on deletes. If you prefer to abort th
 when an attempt to delete an unknown id, use `.option("ignoreUnknownIds", "true")`
 for those resources types.
 
+## Asset hierarchy (beta)
```

Comment:
I don't exactly know why, but the title "Asset hierarchy" feels a bit off to me. Maybe "Asset hierarchies" or "Asset hierarchy ingestion"? Feel free to ignore this, as it's only my gut feeling :)

Reply:
Will go for builder here as well